app/Console/Commands/sendMailSMTP.php

```diff
@@ -39,6 +39,14 @@ public function __construct()
      */
     public function handle()
     {
-        Mail::to('saul@hudson.com')->send(new maiLTest());
+        //Send mail
+        $to_name = '';
+        $to_email = '';
+        $data = array();
+
+        Mail::send('mails.example', $data, function($message) use ($to_name, $to_email) {
+            $message->to($to_email, $to_name)->subject('Artisans Web Testing Mail');
+            $message->from(env('MAIL_FROM_ADDRESS'), env('MAIL_FROM_NAME'));
+        });
```

Comment:
Aqui está um email fixo, isso precisa de vir do json de configurações.

Essa classe ja precisa estar integrada com o sistema de configurações.

Reply:
Verdade @gilbertoss 
@DavidLordCaos command é para criação de comandos para o artisan.

O que precisamos é uma classe que dispara email a partir dos parametros que recebe. Esse parametros serão enviados pelo webhook.

@gilbertoss tem uma classe para o David ver de exemplo. Estou sentindo falta de uma documentação para quem for apoiar o projeto saber como cria outras integrações.